tailwind.config.js

```diff
@@ -45,22 +45,79 @@ export default {
           800: 'var(--accent-800)',
           900: 'var(--accent-900)',
         },
+        'brand-purple': '#491d8d',
+        'brand-blue': '#42afc5',
+        'black': '#000000',
+        'white': '#ffffff',
+        'slate-400': '#94a3b8',
+        'slate-500': '#64748b',
+      },
+      backgroundImage: {
+        'brand-gradient-primary': 'linear-gradient(to right, rgb(73, 29, 141), rgb(66, 176, 197))',
+        'brand-gradient-secondary': 'linear-gradient(to right, rgb(73, 29, 141), rgb(102, 136, 207))',
+        'brand-gradient-white-purple': 'linear-gradient(to right, #ffffff, #491d8d)',
+        'brand-gradient-white-blue': 'linear-gradient(to right, #ffffff, #42afc5)',
```

Comment:
Are these being used? Same question for the remaining settings such as keyframes and animation below.

Reply:
these were added when I tested the other spinners that used Tailwind, but now since we're using just one spinner with scss, we can remove these.